apps/docs/pages/guides/ai/going-to-prod.mdx

```diff
@@ -0,0 +1,100 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'ai-going-to-prod',
+  title: 'Going to Production',
+  description: 'Checklist for going to production with your AI application.',
+  subtitle: 'Going to production checklist for AI applications.',
+  sidebar_label: 'Going to Production',
+}
+
+This guide will help you to prepare your application for production. We'll provide actionable steps to help you scale your application, ensure that it is reliable, can handle the load, and provide optimal precision for your use case.
+
+See our [Engineering for Scale](/docs/guides/ai/engineering-for-scale) guide for more information about engineering at scale.
+
+## Do you need indexes?
+
+Sequential scans will result in significantly higher latencies and lower throughput, guaranteeing 100% precision and not being RAM bound.
+
+There are a couple of cases where you might not need indexes:
+
+- You have a small dataset and don't need to scale it.
+- You are not expecting high amounts of vector search queries per second.
+- You need to guarantee 100% precision.
+
+You don't have to create indexes in these cases and can use sequential scans instead. This type of workload will not be RAM bound and will not require any additional resources but will result in higher latencies and lower throughput. Extra CPU cores may help to improve queries per second, but it will not help to improve latency.
+
+On the other hand, if you need to scale your application, you will need to create indexes. This will result in lower latencies and higher throughput, but will require additional RAM to make use of Postgres Caching. Also, using indexes will result in lower precision, as with indexes, you replace exact (KNN) search with approximate (ANN) search.
```

Comment:
```suggestion
On the other hand, if you need to scale your application, you will need to create indexes. This will result in lower latencies and higher throughput, but will require additional RAM to make use of Postgres Caching. Also, using indexes will result in lower precision, since you are replacing exact (KNN) search with approximate (ANN) search.
```